.github/pull_request_template.md

```diff
@@ -1,23 +1,24 @@
+<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
 ## Summary of the Pull Request
 
-**What is this about:**
+<!-- Please review the items on the PR checklist before submitting-->
+## PR Checklist
 
-**What is included in the PR:** 
-
-**How does someone test / validate:** 
-
-## Quality Checklist
-
-- [ ] **Linked issue:** #xxx
-- [ ] **Communication:** I've discussed this with core contributors in the issue. 
+- [ ] **Closes:** #xxx
+- [ ] **Communication:** I've discussed this with core contributors already. If work hasn't been agreed, this work might be rejected
 - [ ] **Tests:** Added/updated and all pass
-- [ ] **Installer:** Added/updated and all pass
 - [ ] **Localization:** All end user facing strings can be localized
-- [ ] **Docs:** Added/ updated
-- [ ] **Binaries:** Any new files are added to WXS / YML
-   - [ ] No new binaries
-   - [ ] [YML for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/pipeline.user.windows.yml#L68) for new binaries
-   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries
+- [ ] **Dev docs:** Added/updated
+- [ ] **New binaries:** Added on the required places
+   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
+   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
+   - [ ] [YML for CI pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects
+   - [ ] [YML for signed pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/release.yml)
```

Comment:
What is the difference between signing json and signing yml? Can we describe it a bit?

Reply:
One is for the CI, other is for what does the final release. Chances are honestly very few Will other touch these two